test/Main.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Main where
+
+import GitHub.Data.Name
+import Hit.Issue (parseOwnerRepo)
+import Test.Hspec
+
+
+main :: IO ()
+main = hspec $ do
+  describe "parseOwnerRepo" $ do
+    let expectedOwnerName = N "kowainik"
+    let expectedRepoName = N "hit-on"
+    it "Parses a GitHub repo link (SSH) and gives use the Owner Name and Repo Name separately" $ do
+      parseOwnerRepo "git@github.com:kowainik/hit-on.git" `shouldBe` (Just (expectedOwnerName, expectedRepoName))
+
+    it "Parses a GitHub repo link (SSH) and gives use the Owner Name and Repo Name separately" $ do
+      parseOwnerRepo "git@github.com:kowainik/hit-on" `shouldBe` (Just (expectedOwnerName, expectedRepoName))
+
+
+    it "Parses a GitHub repo link (HTTPS) and gives use the Owner Name and Repo Name separately" $ do
+      parseOwnerRepo "https://github.com/kowainik/hit-on.git" `shouldBe` (Just (expectedOwnerName, expectedRepoName))
+
+
+    it "Parses a GitHub repo link (HTTPS) and gives use the Owner Name and Repo Name separately" $ do
```

Comment:
You've tested _happy paths_ -- when function works. But usually, tests should include cases where a function can't return value. In our case you can add couple tests where `parseOwnerRepo` returns `Nothing`. Just to increase test coverage.

Reply:
This is an important thing, which I missed out... I'll include these things. Thanks a lot for the detailed feedback!

Reply:
I've added 4 more cases of incorrect/broken links that are expected to return `Nothing`. 
Do let me know if you can think of more test cases. I'll add them! 😄